.vscode/launch.json

```diff
@@ -0,0 +1,53 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "interactive",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
+            "program": "${workspaceFolder}/cmd/launcher",
+            "args": [
+                "interactive",
+                "--osqueryd_path=${workspaceFolder}/debug/bin/osqueryd",
+                "--osquery_flag=allow-unsafe",
+                "--osquery_flag=verbose",
+                // add atc config to the path below to load atc config
+                "--osquery_flag=config_path=${workspaceFolder}/debug/kolide-atc-config.json",
+            ],
+            "console": "integratedTerminal",
+            "asRoot": true,
+        },
+        {
+            "name": "k2",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
+            "program": "${workspaceFolder}/cmd/launcher",
+            "args": [
+                "--debug",
+                "--hostname=localhost:3443",
+                "--control",
+                "--control_hostname=localhost:3443",
+                "--enroll_secret_path=${workspaceFolder}/k2_enroll_secret",
```

Comment:
This all feels a little weird to me. But I'm also not sure what would be cleaner. Maybe we should ship a script to do this as a one-time setup, and then conditionalize the vscode tooling on whether it's been setup?